--- origsrc/QuiteRSS-0.10.1-src/QuiteRSS.pro	2012-08-16 00:45:23.000000000 -0500
+++ src/QuiteRSS-0.10.1-src/QuiteRSS.pro	2012-09-01 23:10:15.335468200 -0500
@@ -131,7 +131,7 @@ unix {
   icon_48.path =  $$PREFIX/share/icons/hicolor/48x48/apps
   icon_64.path =  $$PREFIX/share/icons/hicolor/64x64/apps
 
-  translations.files = $$DESTDIR/lang/*.qm
+  translations.files = $$OUT_PWD/$$DESTDIR/lang/*.qm
   translations.path =  $$PREFIX/share/quiterss/lang
   translations.CONFIG += no_check_exist
 
