--- origsrc/QuiteRSS-0.13.3-src/QuiteRSS.pro	2013-08-30 11:46:17.000000000 -0500
+++ src/QuiteRSS-0.13.3-src/QuiteRSS.pro	2013-11-14 18:40:40.187880100 -0600
@@ -252,7 +252,7 @@ unix {
   icon_128.path = $$quote($$PREFIX/share/icons/hicolor/128x128/apps)
   icon_256.path = $$quote($$PREFIX/share/icons/hicolor/256x256/apps)
 
-  translations.files = $$quote($$DESTDIR/lang/)*.qm
+  translations.files = $$quote($$OUT_PWD/$$DESTDIR/lang/)*.qm
   translations.path =  $$quote($$DATA_DIR/lang)
   translations.CONFIG += no_check_exist
 
